### PR TITLE
Add url override feature

### PR DESCRIPTION
**Pull request by drnluz**

### CODE REVIEW

File: lib/solr/request/runner.rb [hunk 1]
```diff
@@ -52,7 +52,9 @@ def call
 
       def solr_urls
         @solr_urls ||= begin
-          urls = if Solr.cloud_enabled? || Solr.master_slave_enabled?
+          urls = if Solr.node_url_override
+            [File.join(Solr.node_url_override, collection_name.to_s)]
```
Comment (route) [inline lib/solr/request/runner.rb]: We better use addressable here, seems to be logically more correct

**Comment by route** [comment]
LGTM

**Comment by SeTeM** [comment]
Could you mention this feature in readme, please?

File: lib/solr/request/runner.rb (hunk 1 above)
Comment (drnluz) [reply lib/solr/request/runner.rb]: yeah but the issue is that Addressable::URI.join will remove the path from the base url case it exists. For example
```
[3] pry(main)> Addressable::URI.join('http://localhost:8983/solr', 'test-core').to_s
=> "http://localhost:8983/test-core"
```

Comment (route) [reply lib/solr/request/runner.rb]: What about parse and then join?

Comment (drnluz) [reply lib/solr/request/runner.rb]: same thing
```
[5] pry(main)> Addressable::URI.parse('http://localhost:8983/solr').join('/en')
=> #<Addressable::URI:0x3fca81816bf0 URI:http://localhost:8983/en>
```
unless we parse and then manipulate the parsed path, but it seems too much work for a just a  string concatenation.

**Comment by drnluz** [comment]
merging to be able to use it on machinio. @route let me know what you think about the `File.join` (my last comment). I can handle it separately based on what we decide to do there.

File: lib/solr/request/runner.rb (hunk 1 above)
Comment (route) [reply lib/solr/request/runner.rb]: weird, ok then

**Pull request merged**